x-pack/plugins/alerting/public/pages/maintenance_windows/translations.ts

```diff
@@ -568,3 +568,24 @@ export const EXPERIMENTAL_DESCRIPTION = i18n.translate(
 export const UPCOMING = i18n.translate('xpack.alerting.maintenanceWindows.upcoming', {
   defaultMessage: 'Upcoming',
 });
+
+export const UPGRADE_TO_PLATINUM = i18n.translate(
+  'xpack.alerting.maintenanceWindows.callout.updateToPlatinumTitle',
+  {
+    defaultMessage: 'Upgrade to an appropriate license',
+  }
+);
+
+export const LINK_APPROPRIATE_LICENSE = i18n.translate(
+  'xpack.alerting.maintenanceWindows.callout.appropriateLicense',
+  {
+    defaultMessage: 'appropriate license',
```

Comment:
Since we're linking to the subscriptions page, I think using "subscription level" is more appropriate here:
```suggestion
    defaultMessage: 'subscription level',
```